docs/index.md

```diff
@@ -251,7 +251,7 @@ Example:
 def jwt_response_payload_handler(token, user=None, request=None):
     return {
         'token': token,
-        'user': UserSerializer(user).data
+        'user': UserSerializer(user,context={'request': request},).data
```

Comment:
Small nitpick in all three cases. `UserSerializer(user, context={'request': request}).data`